packages/components/src/vue/components/morph-parts/principal-parts.vue

```diff
@@ -40,20 +41,20 @@
             :type="types.pronunciation"
         />
 
-        <span class="feature_extras" v-if="hasExtras" >
+        <div class="feature_extras" v-if="hasExtras" >
```

Comment:
is this changing them from displaying inline to block?

Reply:
Because `span` is an inline element and we cannot use margins for it (which we need to set spacing). We're also using `display: flex` for a parent element, and that works better when its children are block-level elements. I just think `display: flex` is so powerful (can set spacing an alignment of elements in both directions in many different ways) that we should use it whenever possible (Bootstrap recognized that by switching to `dispaly: flex` in its latest version).